web/frontend/src/components/ResultsDisplay.tsx

```
@@ -36,35 +50,110 @@ const ResultsDisplay: React.FC<ResultsDisplayProps> = ({ results, showDetailedRe
+        await handleClipboardShare(shareText);
+      } catch (clipboardErr) {
+        console.error('All sharing methods failed:', clipboardErr);
+        alert('Unable to share idea. Please try copying manually.');
+      }
+    }
+  };
+
+  const handleClipboardShare = async (text: string) => {
+    try {
+      // Check if clipboard API is available
+      if (navigator.clipboard && navigator.clipboard.writeText) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)